src/views/AmalgamationSelection.vue

```diff
@@ -0,0 +1,265 @@
+              <v-btn
+                disabled
+                color="primary"
+                large
+              >
+                <strong>Start Vertical Short-form</strong>
+              </v-btn>
+            </div>
+          </v-card>
+        </v-col>
+
+        <!-- Third Choice -->
+        <v-col>
+          <v-card
+            flat
+            class="pt-6 pb-8 px-4"
+          >
+            <h2>
+              Regular long-form amalgamation
+            </h2>
+            <br>
+            <p>
+              If amalgamating corporations don't meet criteria for a short-form amalgamation, they
+              need to complete the following steps to amalgamate:
+            </p>
+            <ul>
+              <li>The shareholders of each amalgamating corporation must approve the amalgamation</li>
+              <li>The amalgamating corporations must enter into an amalgamation agreement</li>
+              <li>
+                The amalgamating corporations then file articles of amalgamation along with the
+                amalgamation agreement to obtain a certificate of amalgamation
+              </li>
+            </ul>
+            <br>
+            <p>
+              When the amalgamation is complete, your company will be a
+              <strong>{{ getRegularAmalgamationText() }}.</strong>
+            </p>
+            <div class="start-horizontal-short-form-btn-div">
+              <v-btn
+                color="primary"
+                large
+                @click="startRegularAmalgamation()"
+              >
+                <strong>Start Regular Long-form</strong>
+              </v-btn>
+            </div>
+          </v-card>
+        </v-col>
+      </v-row>
+    </v-container>
+  </div>
+</template>
+
+<script lang="ts">
+import { useBusinessStore } from '@/stores'
+import { Getter } from 'pinia-class'
+import { Component, Vue } from 'vue-property-decorator'
+import Axios from 'axios'
+import { addAxiosInterceptors } from 'sbc-common-components/src/util/interceptors'
+import { CorpTypeCd } from '@bcrs-shared-components/corp-type-module'
+import { AmalgamationTypes, FilingTypes } from '@bcrs-shared-components/enums'
+
+@Component({})
+export default class AmalgamationSelection extends Vue {
+  @Getter(useBusinessStore) getLegalType!: string
+  @Getter(useBusinessStore) isBComp!: boolean
+  @Getter(useBusinessStore) isBcCompany!: boolean
+  @Getter(useBusinessStore) isCcc!: boolean
+  @Getter(useBusinessStore) isUlc!: boolean
+
+  /** Get the amalmagated company result name depending on the type. */
+  getRegularAmalgamationText (): string {
+    if (this.isBComp || this.isBcCompany) return 'BC limited company'
+    if (this.isCcc) return 'BC community contribution company'
+    if (this.isUlc) return 'BC unlimited liability company'
+  }
+
+  /** Start Regular Long-form button pressed. */
+  async startRegularAmalgamation (): Promise<any> {
```

Comment:
Creation of draft and redirecting is a WIP.

Reply:
I suggest adding a window.alert (or just alert) if someone clicks on the button anyway. Currently it does nothing, which is confusing.

Reply:
Fixed, thanks.